docs/src/main/sphinx/admin/properties-http-server.md

```diff
@@ -0,0 +1,155 @@
+# HTTP server properties
+
+HTTP server properties allow you to configure the HTTP server of Trino that
+handles [](/security) including [](/security/internal-communication),  and
+serves the [](/admin/web-interface) and the [client
+API](/develop/client-protocol).
+
+## General
+
+### `http-server.process-forwarded`
+
+- **Type:** [](prop-type-boolean)
+- **Default value:** `false`
+
+Enable treating forwarded HTTPS requests over HTTP as secure. Requires the
+`X-Forwarded-Proto` header to be set to `HTTPS` on forwarded requests. This is
```

Comment:
IIRC this is handled by Jetty and I think Jetty supports other headers also

Reply:
Ok .. do you think we need to change anything on the docs here though?

Also @wendigo ?

Reply:
@mosabua I would say the `X-Forwarded` headers maybe link to this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-For#see_also

Reply:
Done